### ISSUE TITLE
validate_argspec is not a valid play attribute

### ISSUE THREAD

**Issue by sscheib** [ansible/ansible-lint#5168]
##### Summary

`validate_argspec` which was introduced as *Tech Preview* with Core 2.20 (see https://docs.ansible.com/projects/ansible/latest/playbook_guide/playbooks_variables_validation.html) is currently flagged by ansible-lint as invalid keyword.

I am not sure, if the exclude was on purpose (since it's a Tech Preview feature).
If not, I'd give a shot myself with a PR, by adding it to
https://raw.githubusercontent.com/ansible/ansible-lint/main/src/ansiblelint/schemas/playbook.json

Before I raise a PR, however, I wanted to see if it was a deliberate decision to not include it (yet).

I'd appreciate if a maintainer could let me know if it's worth raising a PR for it.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 26.8.0 using ansible-core:2.21.3 ansible-compat:26.8.0 ruamel-yaml:0.19.1 ruamel-yaml-clib:0.2.15
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

Minimal playbook to reproduce:
```yaml
- name: 'Minimal reproducer'
  hosts: 'localhost'
  gather_facts: false
  validate_argspec: true
  tasks:
    - name: 'Test'
      ansible.builtin.debug:
        msg: 'Test'
```

The following error is observed when running ansible-lint:
```console
ansible-lint
  schema[playbook]: $[0] 'gather_facts', 'hosts', 'tasks', 'validate_argspec' do not match any of the regexes: '^(ansible\.builtin\.)?import_playbook$', 'name', 'tags', 'vars', 'when'
Failed: 1 failure(s), 0 warning(s) in 1 files processed of 1 encountered.
```

Core accepts the playbook, however. It also enforces the spec:
```yaml
argument_specs:
  Minimal reproducer:
    options:
      test:
        type: 'str'
        required: true
```

```console
  $ ansible-playbook -i localhost, playbook.yml -e test=foo
  localhost : ok=2  changed=0  unreachable=0  failed=0

  $ ansible-playbook -i localhost, playbook.yml
  missing required arguments: test
```

##### Desired Behavior

`validate_argspec` is recognized as a valid play attribute

##### Actual Behavior

The following error is observed when running ansible-lint:
```console
ansible-lint
  schema[playbook]: $[0] 'gather_facts', 'hosts', 'tasks', 'validate_argspec' do not match any of the regexes: '^(ansible\.builtin\.)?import_playbook$', 'name', 'tags', 'vars', 'when'
Failed: 1 failure(s), 0 warning(s) in 1 files processed of 1 encountered.
```
